### ISSUE TITLE
Rename "master" to "main"

### ISSUE THREAD

**Issue by JStech** [ros-planning/moveit_calibration#22]
Since this is being discussed on the [main MoveIt repo](https://github.com/ros-planning/moveit/issues/2213), I wanted to bring it up here, too. I think whatever is done there should also be mirrored here--no reason for inconsistency amongst MoveIt-branded software.

I'll add my personal opinion: I think it's a small change, both in terms of the effort required and the benefit gained. It won't magically fix the myriad of thorny issues related to inclusion and representation in robotics (at least in the US), but it is a step in the right direction.

**Comment by christian-rauch** [comment]
Wouldn't this break anyone's local git repo as `origin/master` would just disappear? If a `main` branch is really required, it would make sense to have it in parallel to the `master` branch and remove the `master` branch in the repo when no-one cloned from it for ~6 months or so.

**Comment by mamoll** [comment]
I don't think it would break things if [this page](https://github.com/github/renaming) is correct, but perhaps worth testing in a dummy repo to make sure.

**Comment by christian-rauch** [comment]
Ok, so it will change the references to `master` in the web interface and print warnings for CLI users. In this case, a local or forked repo still has to update the branches manually. It will not forward pushed from a local `master` to `origin/main` as far as I understand. So better test this before breaking many local repos.

**Comment by JStech** [comment]
As this is happening (or already happened) for many repos on github and elsewhere, and has even caught [some press](https://www.zdnet.com/article/github-to-replace-master-with-main-starting-next-month/), I'm not too worried that we'd hit some new snag in the process that hasn't already been dealt with.

Thanks for bringing this back up, though. The main MoveIt repo will continue using `master`, but MoveIt2 is already using `main`. I think we should make the switch, to match the direction of the MoveIt community. I also think it's a good change in and of itself.

I'll read up on how to make the change. I'll post to this issue again before actually doing anything, though.
